distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -36,7 +36,27 @@ type Type
 
     ## ADVANCED
 
-       Returns a vector of `Meta.Constructor` for this type
+       Returns a vector of method names that can be invoked
+       on instances of this type.
+
+
+
+       To obtain list of _static methods_ on a given type
+       use `Meta.type_of`.
+
+       > Example
+         All instance methods to invoke on `Integer` as
+         `(v:Integer) v.method_name...`:
```

Comment:
What is this `...` syntax here? I don't think this is valid Enso code.

Reply:
Following code is valid and prints `5`:
```haskell
from Standard.Base import Integer
foo (x:Integer) = x.abs...
main = foo -5
```